index.js

```diff
@@ -87,6 +84,10 @@ app.get('/api/getVideoSponsorTimes', function (req, res) {
     });
 });
 
+function GetIP (req) {
+    return BehindProxy ? req.headers['x-forwarded-for'] || req.connection.remoteAddress;
```

Comment:
Great idea. I'll be able to make `BehindProxy` and `globalSalt` be set in a separate git ignored config file to make it easier to setup a server for others.

I do plan on making a "Read only" server soon that will forward submission to the main server but make it so you don't have to send every get sponsor request to the main server.

Reply:
Seems like serverless would be better if you start doing that :/

Reply:
Serverless solves none of these problems

Reply:
"making a "Read only" server soon that will forward submission to the main server"
On serverless this would no longer be needed